vip-go-wpcom-compat.php

```diff
@@ -9,3 +9,4 @@
 
 require_once __DIR__ . '/wpcom-deprecated-functions.php';
 require_once __DIR__ . '/wpcom-shortcodes.php';
+require_once __DIR__ . '/wpcom-make-content-clickable.php';
```

Comment:
Instead of a single file for a single filter, we should probably switch this to something like `wpcom-hooks` where we can add future filters/actions as well.